.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
The dockerfile has an npm install step to install the deps according to the node version in the container. node_modules is there in the dockerignore to prevent the node modules from the host getting copied over inside the container. That could cause weird behaviour if, for example, if you had system/node version specific deps on the host